tests/e2e/project/commit.spec.ts

```diff
@@ -81,6 +82,19 @@ test("deleted file", async ({ page }) => {
   await expect(page.locator("text=.hidden deleted")).toBeVisible();
 });
 
+test("moved file", async ({ page }) => {
+  await page.goto(`${sourceBrowsingUrl}/commits/${aliceMainHead}`);
+  await expect(
+    page.locator("header").filter({ hasText: "bin/true -> src/true moved" }),
+  ).toBeVisible();
+});
+
+test.fixme("copied file", async () => {
+  console.log(
+    "Currently not tested, since seems not possible from radicle-surf",
+  );
+});
+
```

Comment:
I wrote it as a placeholder, since I think we will support copies detection for files where the original file has been modified in the same changeset.
https://radicle.zulipchat.com/#narrow/stream/369277-heartwood/topic/detecting.20copies.20in.20radicle-surf/near/369635134

Reply:
Removed it and left only a TODO comment so I can find it quickly once https://github.com/radicle-dev/radicle-git/pull/131 gets merged